src/main/java/org/opensearch/flowframework/workflow/CreateIndexStep.java

```diff
@@ -96,15 +119,132 @@ public String getName() {
         return NAME;
     }
 
+    /**
```

Comment:
It's fine to keep the above CreateIndexRequest code (L82:86) here and later we can follow the same path like ml-commons for CreateIndexStep to create an index.